src/components/LoginForm..jsx

```diff
@@ -0,0 +1,100 @@
+import React, { useState } from 'react';
+import {Tab, Tabs, TextField, Stack, Button, Container } from "@mui/material";
+import { Box } from '@mui/system';
+
+const passwordMatchingText = {
+    error: "Passwords do not match",
+    valid: ""
+}
+function LoginForm() {
+    const [currentTabIndex, setCurrentTabIndex] = useState(0);
+    const [email, setEmail] = useState('');
+    const [password, setPassword] = useState('');
+    const [passwordVerification, setPasswordVerification] = useState('');
+    const [passwordMatchingMsg, setPasswordMatchingMsg] = useState(passwordMatchingText.valid)
+    
+    const handleTabChange = (event, tabIndex) => {
+        setCurrentTabIndex(tabIndex);
+        setEmail('')
+        setPassword('')
+        setPasswordVerification('')
+        setPasswordMatchingMsg(passwordMatchingText.valid)
+    };
+
+    const handleEmailChange = (event) => {
+        setEmail(event.target.value)
+    }
+
+    const handlePasswordChange = (event) => {
+        setPassword(event.target.value)
+        setPasswordMatchingMsg(passwordMatchingText.valid)
+    }
+
+    const handlePasswordVerificationChange = (event) => {
+        setPasswordVerification(event.target.value)
+        setPasswordMatchingMsg(passwordMatchingText.valid)
+    }
+
+    /*
+    TODO:
+    - Handle api status code (200 vs != 200)
+    */
+    const onLogin = (event) => {
+        event.preventDefault()
+        alert("Welcome " + email)
+
+    }
+
+    /*
+    TODO:
+    - Handle api status code (200 vs != 200)
+    */
+    const onCreateAccount = (event) => {
+        event.preventDefault()
+        if (password !== passwordVerification) {
+            setPasswordMatchingMsg(passwordMatchingText.error)
+        }
+    }
+
+    return (
+        <Container sx={{width: "25%",border: "1px solid gray" }}>
```

Comment:
I'm not familiar with Material UI but is it possible to set breakpoints to adjust the layout as the viewport width narrows? Here's what happens when I narrow the browser window:

<img width="1935" alt="Screenshot 2023-10-29 at 1 39 40 PM" src="https://github.com/secure-password-manager/PasswordManagerWeb/assets/39425112/e365daaa-ba9f-44d7-bd26-611705b1b474">

<img width="1501" alt="Screenshot 2023-10-29 at 1 39 53 PM" src="https://github.com/secure-password-manager/PasswordManagerWeb/assets/39425112/299c42e1-5c13-4cc4-835e-c6a7f8539302">

<img width="570" alt="Screenshot 2023-10-29 at 1 40 06 PM" src="https://github.com/secure-password-manager/PasswordManagerWeb/assets/39425112/e26c31c1-e106-4789-93a6-6b3cf2a2213f">